todo_app/templates/index.html

```diff
@@ -5,13 +5,21 @@
   <div class="jumbotron">
     <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
+    <form action = "http://localhost:5000" method = "post">
```

Comment:
`action` should either be a relative path (i.e. `action="/"`) or it should use `{{url_for('index')}}`

Hardcoding to localhsot:5000 means this will break if you try to deploy to any server.